src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.ILLink.targets

```
@@ -141,6 +142,8 @@ Copyright (c) .NET Foundation. All rights reserved.
     <PropertyGroup>
       <_ExtraTrimmerArgs>--skip-unresolved true $(_ExtraTrimmerArgs)</_ExtraTrimmerArgs>
       <TrimMode Condition=" '$(TrimMode)' == '' ">copyused</TrimMode>
+      <_TrimmerLinkSymbols>$(DebuggerSupport)</_TrimmerLinkSymbols>
```

I know we consider `_TrimmerLinkSymbols` to be private, but it is safest to check if it is already set and not overwrite the value if someone has already set it.

```suggestion
      <_TrimmerLinkSymbols Condition=" '$(_TrimmerLinkSymbols)' == '' ">$(DebuggerSupport)</_TrimmerLinkSymbols>
```

I could imagine a case where someone wanted .pdbs to be created correctly, but also wanted the "debugger only" code to be trimmed from the application.